README.md

```diff
@@ -8,10 +8,19 @@ This project is in a very early stage and will be both restructured and renamed.
 
 ## Quickstart
 
-to install the python and other developer requirements into a venv run:
+To install the python and other developer requirements into a venv run:
 
     make install
 
+### Configuration options
+
+There's a few env vars that can be used with this project:
+
+* `TEST_TARGET`: Set to `AWS_CLOUD` to use an externally-deployed instance when running tests.
+* `SNAPSHOT_LEGACY_REPORT`: By default set to `0`. Can be set to `1`.
+* `SNAPSHOT_UDPATE`: By default set to `0`. Can be set to `1`.
+* `SNAPSHOT_RAW`: By default set to `0`. Can be set to `1`.
```

Comment:
suggestion: extend with what effect enabling these variables will have

Reply:
I have tried to address your comments in [`c4f0012` (#8)](https://github.com/localstack/localstack-snapshot/pull/8/commits/c4f0012022cc069e7b286ffbfda2da1041d0e6a0), let me know if this is what you meant